src/core/layout/navbar/context.js

```diff
@@ -0,0 +1,11 @@
+import React, { useContext as useReactContext, useState } from 'react'
+
+export const Context = React.createContext({})
+
+export const useContext = () => useReactContext(Context)
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128525072-38e328e3-c50c-4a97-81cd-db5d728feaf0.png)
Let's change this component to use the `classes` props too.

Reply:
Done